### ISSUE TITLE
 Route health update status not correctly 

### ISSUE THREAD

**Issue by duongmn89** [rluisr/mysqlrouter_exporter#28]
Hi, I'm using mysqlrouter 8.0.32 and newest router exporter image and I have a problem
When I turn off mysql backend servers, then the API of the router reports correctly
`curl -k -u username:password https://192.168.1.10:8443/api/20190715/routes/bootstrap_rw/health
{"isAlive":false}`
But the exporter still reports 1 (active)
`mysqlrouter_route_health{name="bootstrap_rw",router_hostname="mysqlrouter-ABCXYZ"} 1
`

My exporter args:
`/app -k -p 9105 --collect.metadata.status "true" --collect.route.connections.byte_from_server "true" --collect.route.connections.byte_to_server "true"  --collect.route.connections.time_started "true" --collect.route.connections.time_connected_to_server "true" --collect.route.connections.time_last_sent_to_server "true" --collect.route.connections.time_received_from_server "true"`
Please take a look

**Comment by rluisr** [comment]
In my production environment using MySQL 8.0.32 InnoDB Cluster, an exporter output seems correct.

```
curl -k -u https://mysql11.luis.local:8443/api/20190715/routes/bootstrap_rw/health
{"isAlive":true}

mysqlrouter_route_health{name="bootstrap_rw",router_hostname="mysql11.luis.local"} 1
```

Is it possible that a temporarily network interruption could have caused this to happen?

**Comment by duongmn89** [comment]
hen my innodb cluster is online, the exporter can show it correctly = 1
`mysqlrouter_route_health{name="bootstrap_rw",router_hostname="mysqlrouter-ABCXYZ"} 1`

But when my innodb cluster is offline, the exporter still shows mysqlrouter_route_health = 1

**Comment by duongmn89** [comment]
Show what I want to say is when innodb cluster is offline, the exporter can update metrics mysqlrouter_metadata_config_nodes correctly, but can not update metric mysqlrouter_route_health

**Comment by rluisr** [comment]
Ah OK, I understood.  I think is a bug on an exporter so I'll fix this or PR is welcome.

**Comment by duongmn89** [comment]
@rluisr Please review my PR: https://github.com/rluisr/mysqlrouter_exporter/pull/29

**Comment by rluisr** [comment]
released v4.0.4

**Comment by duongmn89** [comment]
@rluisr : I tried new released but this bug still exits 
Please help to fix :(
